aws/ebs_test.go

```diff
@@ -0,0 +1,117 @@
+package aws
+
+import (
+	"testing"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+func createTestEBSVolume(t *testing.T, session *session.Session, name string) ec2.Volume {
+	svc := ec2.New(session)
+	volume, err := svc.CreateVolume(&ec2.CreateVolumeInput{
+		AvailabilityZone: awsgo.String("us-west-2a"),
```

Comment:
I just noticed that these tests all hard-code the region. The downside to this is if we have some region-specific bug, we won't catch it.

You can use [this method](https://github.com/gruntwork-io/terratest/blob/master/aws/region.go#L20) to pick a random one.